### DISCUSSION TITLE
Problem with OptionalParametersAfterRequiredRector

### DISCUSSION THREAD

**Discussion by alejomoreira** [rectorphp/rector#7397] [General]
Hey Guys, when Is use the [Rector\CodeQuality\Rector\ClassMethod\OptionalParametersAfterRequiredRector](https://github.com/rectorphp/rector/blob/main/rules/CodeQuality/Rector/ClassMethod/OptionalParametersAfterRequiredRector.php) rule it looks like is not rewriting the call to the method, is there any way to do that, or we should change it manually for each call?
Thanks!

**Comment by TomasVotruba** [comment]
Hi, it should handle the method calls as well: https://github.com/rectorphp/rector-src/blob/main/rules-tests/CodeQuality/Rector/ClassMethod/OptionalParametersAfterRequiredRector/Fixture/update_method_call.php.inc

But possibly it misses some edge-case that is not covered yet. Could you provide test fixture with your use case?
